### PR TITLE
Fix doc warnings

### LINKED ISSUE #157: Address warnings when building docs
When I run the docs command  `python -m etstool docs`, the following warnings are reported. Note that it might not be worth the effort addressing all of the warnings until after other issues in `apptools` are addressed - regarding removal of outdated/unused modules in `apptools`.

<details>

```
reading sources... [100%] undo/Introduction
WARNING: autodoc: failed to import module 'api' from module 'apptools.help.help_plugin'; the following exception was raised:
No module named 'envisage'
WARNING: autodoc: failed to import module 'help_plugin' from module 'apptools.help.help_plugin'; the following exception was raised:
No module named 'envisage'
WARNING: autodoc: failed to import module 'help_submenu_manager' from module 'apptools.help.help_plugin'; the following exception was raised:
No module named 'envisage'
WARNING: autodoc: failed to import module 'demo_action' from module 'apptools.help.help_plugin.action'; the following exception was raised:
No module named 'envisage'
WARNING: autodoc: failed to import module 'doc_action' from module 'apptools.help.help_plugin.action'; the following exception was raised:
No module named 'envisage'
WARNING: autodoc: failed to import module 'example_action' from module 'apptools.help.help_plugin.action'; the following exception was raised:
No module named 'envisage'
WARNING: autodoc: failed to import module 'load_url_action' from module 'apptools.help.help_plugin.action'; the following exception was raised:
No module named 'envisage'
c:\users\rporuri\work\github\ets\apptools\apptools\io\h5\file.py:docstring of apptools.io.h5.file.H5Group.create_array:8: WARNING: Unexpected section title.

Parameters
----------
c:\users\rporuri\work\github\ets\apptools\apptools\io\h5\file.py:docstring of apptools.io.h5.file.H5Group.create_dict:8: WARNING: Unexpected section title.

Parameters
----------
c:\users\rporuri\work\github\ets\apptools\apptools\io\h5\file.py:docstring of apptools.io.h5.file.H5Group.create_group:8: WARNING: Unexpected section title.

Parameters
----------
c:\users\rporuri\work\github\ets\apptools\apptools\io\h5\file.py:docstring of apptools.io.h5.file.H5Group.create_table:8: WARNING: Unexpected section title.

Parameters
----------
c:\users\rporuri\work\github\ets\apptools\apptools\io\h5\file.py:docstring of apptools.io.h5.file.H5Group.remove_group:8: WARNING: Unexpected section title.

Parameters
----------
c:\users\rporuri\work\github\ets\apptools\apptools\io\h5\file.py:docstring of apptools.io.h5.file.H5Group.remove_node:8: WARNING: Unexpected section title.

Parameters
----------
WARNING: autodoc: failed to import module 'logger_plugin' from module 'apptools.logger.plugin'; the following exception was raised:
No module named 'envisage'
WARNING: autodoc: failed to import module 'qt4_widget' from module 'apptools.permissions.adapters'; the following exception was raised:
Cannot import PySide, PySide2, PyQt5 or PyQt4
WARNING: autodoc: failed to import module 'wx_window' from module 'apptools.permissions.adapters'; the following exception was raised:
No module named 'wx'
WARNING: autodoc: failed to import module 'spickle' from module 'apptools.persistence'; the following exception was raised:
This module does not work with Python 3.x
c:\users\rporuri\work\github\ets\apptools\apptools\preferences\scoped_preferences.py:docstring of apptools.preferences.scoped_preferences.ScopedPreferences:13: WARNING: Literal block expected; none found.
WARNING: autodoc: failed to import module 'widget_editor' from module 'apptools.preferences.ui'; the following exception was raised:
No module named 'traitsui.null.editor'
c:\users\rporuri\work\github\ets\apptools\apptools\scripting\recorder.py:docstring of apptools.scripting.recorder:3: WARNING: Unknown directive type "todo".

.. todo:: - Support for dictionaries?
c:\users\rporuri\work\github\ets\apptools\apptools\scripting\recorder.py:docstring of apptools.scripting.recorder.Recorder.record:4: WARNING: Unexpected section title.

Parameters:
-----------
c:\users\rporuri\work\github\ets\apptools\apptools\scripting\recorder.py:docstring of apptools.scripting.recorder.Recorder.register:23: WARNING: Unexpected section title.

Parameters:
-----------
c:\users\rporuri\work\github\ets\apptools\apptools\scripting\util.py:docstring of apptools.scripting.util.start_recording:4: WARNING: Unexpected section title.

Parameters:
-----------
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:2: WARNING: duplicate object description of apptools.selection.errors, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:5: WARNING: duplicate object description of apptools.selection.errors.IDConflictError, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:13: WARNING: duplicate object description of apptools.selection.errors.ListenerNotConnectedError, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:21: WARNING: duplicate object description of apptools.selection.errors.ProviderNotRegisteredError, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:2: WARNING: duplicate object description of apptools.selection.i_selection, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:5: WARNING: duplicate object description of apptools.selection.i_selection.IListSelection, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:13: WARNING: duplicate object description of apptools.selection.i_selection.IListSelection.indices, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:20: WARNING: duplicate object description of apptools.selection.i_selection.IListSelection.items, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:27: WARNING: duplicate object description of apptools.selection.i_selection.ISelection, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:35: WARNING: duplicate object description of apptools.selection.i_selection.ISelection.is_empty, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:41: WARNING: duplicate object description of apptools.selection.i_selection.ISelection.provider_id, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:2: WARNING: duplicate object description of apptools.selection.i_selection_provider, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:5: WARNING: duplicate object description of apptools.selection.i_selection_provider.ISelectionProvider, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:13: WARNING: duplicate object description of apptools.selection.i_selection_provider.ISelectionProvider.get_selection, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:24: WARNING: duplicate object description of apptools.selection.i_selection_provider.ISelectionProvider.provider_id, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:31: WARNING: duplicate object description of apptools.selection.i_selection_provider.ISelectionProvider.selection, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:39: WARNING: duplicate object description of apptools.selection.i_selection_provider.ISelectionProvider.set_selection, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:2: WARNING: duplicate object description of apptools.selection.list_selection, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:5: WARNING: duplicate object description of apptools.selection.list_selection.ListSelection, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:16: WARNING: duplicate object description of apptools.selection.list_selection.ListSelection.from_available_items, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:32: WARNING: duplicate object description of apptools.selection.list_selection.ListSelection.indices, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:39: WARNING: duplicate object description of apptools.selection.list_selection.ListSelection.is_empty, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:45: WARNING: duplicate object description of apptools.selection.list_selection.ListSelection.items, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:52: WARNING: duplicate object description of apptools.selection.list_selection.ListSelection.provider_id, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:2: WARNING: duplicate object description of apptools.selection.selection_service, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:5: WARNING: duplicate object description of apptools.selection.selection_service.SelectionService, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:20: WARNING: duplicate object description of apptools.selection.selection_service.SelectionService.add_selection_provider, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:32: WARNING: duplicate object description of apptools.selection.selection_service.SelectionService.connect_selection_listener, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:54: WARNING: duplicate object description of apptools.selection.selection_service.SelectionService.disconnect_selection_listener, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:64: WARNING: duplicate object description of apptools.selection.selection_service.SelectionService.get_selection, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:80: WARNING: duplicate object description of apptools.selection.selection_service.SelectionService.has_selection_provider, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:86: WARNING: duplicate object description of apptools.selection.selection_service.SelectionService.remove_selection_provider, other instance in selection/selection, use :noindex: for one of them
C:\Users\rporuri\work\github\ets\apptools\docs\source\api\apptools.selection.rst:97: WARNING: duplicate object description of apptools.selection.selection_service.SelectionService.set_selection, other instance in selection/selection, use :noindex: for one of them
WARNING: autodoc: failed to import module 'api' from module 'apptools.template'; the following exception was raised:
Traceback (most recent call last):
  File "C:\Users\rporuri\.edm\envs\apptools-test-3.6\lib\site-packages\sphinx\ext\autodoc\importer.py", line 32, in import_module
    return importlib.import_module(modname)
  File "C:\Users\rporuri\.edm\envs\apptools-test-3.6\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "c:\users\rporuri\work\github\ets\apptools\apptools\template\api.py", line 36, in <module>
    from .template_data_names \
  File "c:\users\rporuri\work\github\ets\apptools\apptools\template\template_data_names.py", line 87, in <module>
    selection_bg_color = None
  File "C:\Users\rporuri\.edm\envs\apptools-test-3.6\lib\site-packages\traitsui\editor_factory.py", line 89, in __init__
    HasPrivateTraits.__init__(self, **traits)
  File "C:\Users\rporuri\.edm\envs\apptools-test-3.6\lib\site-packages\traits\base_trait_handler.py", line 75, in error
    object, name, self.full_info(object, name, value), value
traits.trait_errors.TraitError: The 'selection_bg_color' trait of a ToolkitEditorFactory instance must be an integer which in hex is of the form 0xRRGGBB, where RR is red, GG is green, and BB is blue or 'aquamarine' or 'black' or 'blue violet' or 'blue' or 'brown' or 'cadet blue' or 'coral' or 'cornflower blue' or 'cyan' or 'dark green' or 'dark grey' or 'dark olive green' or 'dark orchid' or 'dark slate blue' or 'dark slate grey' or 'dark turquoise' or 'dim grey' or 'firebrick' or 'forest green' or 'gold' or 'goldenrod' or 'green yellow' or 'green' or 'grey' or 'indian red' or 'khaki' or 'light blue' or 'light grey' or 'light steel' or 'lime green' or 'magenta' or 'maroon' or 'medium aquamarine' or 'medium blue' or 'medium forest green' or 'medium goldenrod' or 'medium orchid' or 'medium sea green' or 'medium slate blue' or 'medium spring green' or 'medium turquoise' or 'medium violet red' or 'midnight blue' or 'navy' or 'orange red' or 'orange' or 'orchid' or 'pale green' or 'pink' or 'plum' or 'purple' or 'red' or 'salmon' or 'sea green' or 'sienna' or 'sky blue' or 'slate blue' or 'spring green' or 'steel blue' or 'tan' or 'thistle' or 'turquoise' or 'violet red' or 'violet' or 'wheat' or 'white' or 'yellow green' or 'yellow', but a value of None <class 'NoneType'> was specified.

WARNING: autodoc: failed to import module 'template_data_names' from module 'apptools.template'; the following exception was raised:
Traceback (most recent call last):
  File "C:\Users\rporuri\.edm\envs\apptools-test-3.6\lib\site-packages\sphinx\ext\autodoc\importer.py", line 32, in import_module
    return importlib.import_module(modname)
  File "C:\Users\rporuri\.edm\envs\apptools-test-3.6\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "c:\users\rporuri\work\github\ets\apptools\apptools\template\template_data_names.py", line 87, in <module>
    selection_bg_color = None
  File "C:\Users\rporuri\.edm\envs\apptools-test-3.6\lib\site-packages\traitsui\editor_factory.py", line 89, in __init__
    HasPrivateTraits.__init__(self, **traits)
  File "C:\Users\rporuri\.edm\envs\apptools-test-3.6\lib\site-packages\traits\base_trait_handler.py", line 75, in error
    object, name, self.full_info(object, name, value), value
traits.trait_errors.TraitError: The 'selection_bg_color' trait of a ToolkitEditorFactory instance must be an integer which in hex is of the form 0xRRGGBB, where RR is red, GG is green, and BB is blue or 'aquamarine' or 'black' or 'blue violet' or 'blue' or 'brown' or 'cadet blue' or 'coral' or 'cornflower blue' or 'cyan' or 'dark green' or 'dark grey' or 'dark olive green' or 'dark orchid' or 'dark slate blue' or 'dark slate grey' or 'dark turquoise' or 'dim grey' or 'firebrick' or 'forest green' or 'gold' or 'goldenrod' or 'green yellow' or 'green' or 'grey' or 'indian red' or 'khaki' or 'light blue' or 'light grey' or 'light steel' or 'lime green' or 'magenta' or 'maroon' or 'medium aquamarine' or 'medium blue' or 'medium forest green' or 'medium goldenrod' or 'medium orchid' or 'medium sea green' or 'medium slate blue' or 'medium spring green' or 'medium turquoise' or 'medium violet red' or 'midnight blue' or 'navy' or 'orange red' or 'orange' or 'orchid' or 'pale green' or 'pink' or 'plum' or 'purple' or 'red' or 'salmon' or 'sea green' or 'sienna' or 'sky blue' or 'slate blue' or 'spring green' or 'steel blue' or 'tan' or 'thistle' or 'turquoise' or 'violet red' or 'violet' or 'wheat' or 'white' or 'yellow green' or 'yellow', but a value of None <class 'NoneType'> was specified.

WARNING: autodoc: failed to import module 'i_context_adapter' from module 'apptools.template.impl'; the following exception was raised:
cannot import name 'adapts'
WARNING: autodoc: failed to import module 'enable_editor' from module 'apptools.template.test'; the following exception was raised:
No module named 'wx'
WARNING: autodoc: failed to import module 'scatter_plot' from module 'apptools.template.test'; the following exception was raised:
cannot import name 'Theme'
WARNING: autodoc: failed to import module 'scatter_plot_2' from module 'apptools.template.test'; the following exception was raised:
cannot import name 'Theme'
WARNING: autodoc: failed to import module 'scatter_plot_nm' from module 'apptools.template.test'; the following exception was raised:
cannot import name 'Theme'
WARNING: autodoc: failed to import module 'template_view' from module 'apptools.template.test'; the following exception was raised:
cannot import name 'Theme'
```

</details>

### PR DESCRIPTION
**Pull request by aaronayres35**
fixes #157 

**Checklist**
~- [ ] Add a news fragment if this PR is news-worthy for end users. (see docs/releases/README.rst)~

### CODE REVIEW

**Comment by aaronayres35** [comment]
Currently only the following warnings remain:

```

/Users/aayres/Desktop/apptools/apptools/io/h5/file.py:docstring of apptools.io.h5.file.H5Group.create_array:8: WARNING: Unexpected section title.

Parameters
----------
/Users/aayres/Desktop/apptools/apptools/io/h5/file.py:docstring of apptools.io.h5.file.H5Group.create_dict:8: WARNING: Unexpected section title.

Parameters
----------
/Users/aayres/Desktop/apptools/apptools/io/h5/file.py:docstring of apptools.io.h5.file.H5Group.create_group:8: WARNING: Unexpected section title.

Parameters
----------
/Users/aayres/Desktop/apptools/apptools/io/h5/file.py:docstring of apptools.io.h5.file.H5Group.create_table:8: WARNING: Unexpected section title.

Parameters
----------
/Users/aayres/Desktop/apptools/apptools/io/h5/file.py:docstring of apptools.io.h5.file.H5Group.remove_group:8: WARNING: Unexpected section title.

Parameters
----------
/Users/aayres/Desktop/apptools/apptools/io/h5/file.py:docstring of apptools.io.h5.file.H5Group.remove_node:8: WARNING: Unexpected section title.

Parameters
----------
WARNING: autodoc: failed to import module 'logger_plugin' from module 'apptools.logger.plugin'; the following exception was raised:
No module named 'envisage'

```

The `apptools.io.h5` have been confusing me a bit, and they are directly related to this decorator:
https://github.com/enthought/apptools/blob/2f46fc2c088496db8d48c794c8c79a75c6972613/apptools/io/h5/file.py#L20-L34
Which is used to define the docstrings for the `H5Group` methods.  "Parameters" should be a valid section title, but something is happening when using the docstring from this decorator that is causing problems.

File: apptools/scripting/recorder.py [hunk 1]
```diff
@@ -1,9 +1,6 @@
 """
 Code to support recording to a readable and executable Python script.
 
-TODO:
-    - Support for dictionaries?
```
Comment (rahulporuri) [inline apptools/scripting/recorder.py:5]: it'll be better to replace the TODO, which makes sphinx freak out, with a FIXME/NOTE and keep the comment about supporting dictionaries.

**Comment by rahulporuri** [comment]
@aaronayres35 it looks like the errors in `apptools.io.h5.file.H5Group` are related to bad indentation in the docstring. Here is what I see when i try to print out the docstring for `H5Group.create_array` on the command prompt and note the awkward indentation.

```
>>> help(H5Group.create_array)
Help on function create_array in module apptools.io.h5.file:

create_array(self, node_subpath, array_or_shape, dtype=None, chunked=False, extendable=False, **kwargs)
    ** H5Group wrapper for H5File.create_array: **
    Note that the first argument is a nodepath relative to the group, rather than
    an absolute path. Below is the original docstring:

        Create node to store an array.

            Parameters
            ----------
            node_path : str
                PyTable node path; e.g. '/path/to/node'.
            array_or_shape : array or shape tuple
                Array or shape tuple for an array. If given a shape tuple, the
                `dtype` parameter must also specified.
            dtype : str or numpy.dtype
                Data type of array. Only necessary if `array_or_shape` is a shape.
            chunked : bool
                Controls whether the array is chunked.
            extendable : {None | bool}
                Controls whether the array is extendable.
            kwargs : key/value pairs
                Keyword args passed to PyTables `File.create_(c|e)array`.

>>>
```

**Comment by aaronayres35** [comment]
> @aaronayres35 it looks like the errors in `apptools.io.h5.file.H5Group` are related to bad indentation in the docstring. 

I believe these are fiixed now via use of `inspect.cleandoc()`.  @rahulporuri what should we do about the sole remaining warning about not having `envisage`?
```
WARNING: autodoc: failed to import module 'logger_plugin' from module 'apptools.logger.plugin'; the following exception was raised:
No module named 'envisage'
```

It seems incorrect to need `envisage` to build the docs, I'm not sure what the best workaround for this kind of situation is

**Comment by aaronayres35** [comment]
> @rahulporuri what should we do about the sole remaining warning about not having `envisage`?
> 
> ```
> WARNING: autodoc: failed to import module 'logger_plugin' from module 'apptools.logger.plugin'; the following exception was raised:
> No module named 'envisage'
> ```
> 
> It seems incorrect to need `envisage` to build the docs, I'm not sure what the best workaround for this kind of situation is

The warning is now fixed by the latest commit which tries to import from the `ensiage.api` but if it fails, simply mocks that import so that the docs can still be built.

File: apptools/logger/plugin/logger_plugin.py [hunk 2]
```diff
@@ -15,10 +15,16 @@
...
 import logging
+from unittest.mock import MagicMock
 
 # Enthought library imports.
-from envisage.api import ExtensionPoint, Plugin
+try:
+    from envisage.api import ExtensionPoint, Plugin
+except ModuleNotFoundError:
+    sys.modules['envisage.api'] = MagicMock()
+    from envisage.api import ExtensionPoint, Plugin
```
Comment (kitchoi) [inline apptools/logger/plugin/logger_plugin.py:27]: This affects production code too and this is going to cause strange behaviour that is much harder to debug than getting an ImportError outright.

Mocking for building documentation should only happen within the documentation build workflow. I'd suggest adding this in `docs/source/conf.py`:
```
# Apptools offers an envisage plugin that requires importing from
# envisage.
try:
    import envisage
except ImportError:
    autodoc_mock_imports = [
        'envisage',
    ]

```
and revert this change in `apptools.logger.plugin.logger_plugin`.

File: docs/source/conf.py [hunk 3]
```diff
@@ -15,6 +15,15 @@
...
 
+# Apptools offers an envisage plugin that requires importing from
+# envisage.
+try:
+    import envisage  # noqa
+except ImportError:
+    autodoc_mock_imports = [
+        'envisage',
+    ]
+
```
Comment (kitchoi) [inline docs/source/conf.py:26]: Nit: Although this includes an import and import tends to go to the top, this is still a configuration specific to the autodoc extension.
Normally extension options are placed after "General configuration".
Following current convention, this would go under a section "# Options for autodoc"?

**Review by kitchoi — APPROVED** [review]
LGTM. One nitpick.
